services/user.go

```diff
@@ -0,0 +1,43 @@
+package services
+
+import (
+	"github.com/EloYaniel/academy-go-q42021/apiclient"
+	e "github.com/EloYaniel/academy-go-q42021/entities"
+	repo "github.com/EloYaniel/academy-go-q42021/repositories/contracts"
+)
+
+type UserService struct {
+	repo      repo.UserRepository
+	apiClient apiclient.ApiClient
+}
+
+func NewUserService(repo repo.UserRepository, client apiclient.ApiClient) *UserService {
+	return &UserService{repo: repo, apiClient: client}
+}
+
+func (s *UserService) GetUsers() ([]e.User, error) {
+	users, _ := s.repo.GetUsers()
```

Comment:
Please handle every possible error that could occur, even if it's just logging it